src/dd4pod/python/npsim.py

```diff
@@ -49,14 +49,12 @@ def setupCerenkov(kernel):
   )
   # This could probably be a substring
   SIM.filter.mapDetFilter['DRICH'] = 'opticalphotons'
-  SIM.filter.mapDetFilter['MRICH'] = 'opticalphotons'
-  SIM.filter.mapDetFilter['PFRICH'] = 'opticalphotons'
+  SIM.filter.mapDetFilter['RICHEndcapN'] = 'opticalphotons'
```

Comment:
I think this does a substring match, so just a single line would work too (needs verification):
```
  SIM.filter.mapDetFilter['RICH'] = 'opticalphotons'
```

Reply:
And let's not rely on that

Reply:
Even better this wouldn't be in npsim at all, of course, but I haven't figured out a scalable way of doing that.